quartz/plugins/transformers/links.ts

```diff
@@ -49,6 +53,18 @@ export const CrawlLinks: QuartzTransformerPlugin<Partial<Options> | undefined> =
               allSlugs: ctx.allSlugs,
             }
 
+            // Add frontmatter links to outgoing links
+            if (opts.includeFrontmatterLinks && file.data.frontmatter) {
+              for (const [fmKey, fmValue] of Object.entries(file.data.frontmatter)) {
+                if (fmValue && typeof fmValue === "string") {
```

Comment:
ah you might need to take a look at how we handle links in `ofm.ts` to handle alias, intra-links docs, and marks.

Reply:
Should we do this in ofm instead?

Reply:
> ah you might need to take a look at how we handle links in `ofm.ts` to handle alias, intra-links docs, and marks

I tried to but I didn't manage to get a grasp of how it works. This is still too high level typescript for me :rofl: